fastpass/src/main/scala/scala/meta/internal/fastpass/pantsbuild/commands/SwitchCommand.scala

```diff
@@ -99,7 +99,15 @@ object SwitchCommand extends Command[SwitchOptions]("switch") {
       Files.deleteIfExists(workspaceBloop)
       Files.createSymbolicLink(workspaceBloop, outBloop)
     }
+    runScalafmtSymlink(project, common)
+  }
 
+  def runScalafmtSymlink(
+      project: Project,
+      common: SharedOptions
+  ): Unit = {
```

Comment:
Should this be moved to `SharedCommand`?

Reply:
Good idea! Done